pkg/build/admission/buildpodutil.go

```diff
@@ -56,10 +56,50 @@ func SetBuild(a admission.Attributes, build *buildapi.Build, groupVersion unvers
 	if err != nil {
 		return err
 	}
+
 	err = setBuildInPod(build, pod, groupVersion)
 	if err != nil {
 		return admission.NewForbidden(a, fmt.Errorf("unable to set build in pod: %v", err))
 	}
+
+	err = applyBuildLogLevel(build, pod)
+	if err != nil {
+		return admission.NewForbidden(a, fmt.Errorf("unable to set loglevel in pod: %v", err))
+	}
```

Comment:
shouldn't this be in its own function?  SetBuild is for setting the BUILD env variable w/ the build json definition.